pkgs/servers/filtron/default.nix

```diff
@@ -11,7 +11,7 @@ buildGoModule rec {
     sha256 = "18d3h0i2sfqbc0bjx26jm2n9f37zwp8z9z4wd17sw7nvkfa72a26";
   };
 
-  vendorSha256 = "05q2g591xl08h387mm6njabvki19yih63dfsafgpc9hyk5ydf2n9";
+  vendorSha256 = null; #vendorSha256 = "";
```

Comment:
I have tried with the following hash on my system: `"sha256-yQrXfJkeJnafU9q1YWD0KcS5l5LW1HrQgAjQHlJ5Ahc="` and it seems to build even though I removed go1.17 as you did in this PR

Reply:
The hash isn't reproducible with 1.17.